system/Database/SQLSRV/Forge.php

```diff
@@ -80,7 +84,7 @@ class Forge extends BaseForge
      *
      * @var string
      */
-    protected $createTableIfStr = "IF NOT EXISTS (SELECT * FROM sysobjects WHERE ID = object_id(N'%s') AND OBJECTPROPERTY(id, N'IsUserTable') = 1)\nCREATE TABLE";
+    protected $createTableIfStr = "IF NOT EXISTS(SELECT t.name, s.name as schema_name, t.type_desc FROM sys.tables t INNER JOIN sys.schemas s on s.schema_id = t.schema_id WHERE s.name=N'%s' AND t.name=REPLACE(N'%s', '\"', '') AND t.type_desc='USER_TABLE')\nCREATE TABLE ";
```

Comment:
This is duplicated in lines 109 to 115 below. I'd prefer keeping the one in the constructor though.

Reply:
I agree, it would make sense to just declare the variables without any default value and set them in the constructor.